src/components/calcite-action/calcite-action.tsx

```diff
@@ -119,6 +119,8 @@ export class CalciteAction {
   renderIconContainer(): VNode {
     const { loading } = this;
 
+    const hasIcon = !!this.el.children.length;
```

Comment:
Did Stencil fix this recently? I remember I had an issue with similar logic in Edge.

From https://github.com/Esri/calcite-app-components/pull/785:

> This restores block toggling in Edge. The issue was caused by the 'has content' logic, which caused the block to never open after being closed (`el.children` in Edge includes the elements in the render function, tripping up the logic).

Reply:
dangit. Do you know a better way to figure out if the element has content?

Reply:
Not at the moment. 😢